scripts/main.js

```diff
@@ -0,0 +1,241 @@
+const num = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+let disChar = document.querySelector('.characterdisplay');
```

Comment:
It is better to use the `const` variable declaration mode for this purpose. You may check [this article](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/) for more understanding of and the difference between let, var, and const keywords in JavaScript.